text/0007-weekly-inspo.md

```diff
@@ -0,0 +1,301 @@
+search system using `Next.js`.
+
+### Change Log
+
+It would be nice to keep track of a `CHANGE_LOG` to see the week by week updates that
+happen to the respository. The functionality for this is mostly internal, and can serve
+as a method to expose newer board members to industry practices! We would most likely 
+have a `GitHub` action that runs at the time of dev team meeting each week, that would
+automatically create a pull request to update the `CHANGE_LOG`.
+
+## What are the requirements for this project?
+
+`xplr` will require developing in `Next.js`, writing GitHub Actions, building linters, 
+and deploying to Netlify. Ideally, `xplr` would be built across the CL dev team, but 
+realistically a single person should be able to build out most of the functionality.
+
+TLDR;
+* Learning the `Next.js` framework
+* Learning how to build a linter through `JavaScript`
+* Learning how to utilize GitHub Actions
+
+## What are we launching today?
+
+The entirety of this project seems pretty simple and shouldn't take too long to 
+develop. Future iterations can add custom metadata through the `gray-matter` of
+posts to add images to the metadata and improve SEO. However, everything I laid
+out in the blurb is reasonally achievable.
+
+@sriramb2000 suggested site maps as a means to increase SEO:
+
+https://www.npmjs.com/package/nextjs-sitemap-generator
+
+## FAQ
+
+### How long should posts be?
+
+Inspo posts **should not** take excessive time to write and find. Exploration should
+be fun and hopefully this weekly task doesn't feel like a chore. Finding inspo and
+writing about it should take around 30 minutes. Writing concise documentation is a big
+part of developing and hopefully through this activity, our team will come to 
+appreciate documentation.
+
+### Should posts require code snippets?
+
+The depth of a post should be decided between the reviewer and reviewee during the 
+pull request. If the reviewer thinks the addition of code snippets would improve
+the post itself and the reviewee's understanding of the subject, the reviewer 
+can request the addition.
+
+### When would we start this program?
+
+Ideally, some time before Fall quarter to work out any kinks. Let's tentatively say 
+middle of September.
+
+### Would this program be utilized across all of Creative Labs?
+
+I think it would be nice for everyone on CL to look for inspiration on a daily basis.
+As the future tech director, I only have jurisdiction with my own team. But I would be
+willing to open this up to other teams to use as well.
+
+For example:
+* Design can list all the art/styles/sites that they come across
+* Marketing can list all the concepts/articles/excerpts they read
+
+**Note:** If we are to extend `xplr` to encompass all of CL in the future, it should
+have another RFC to explain the expansion and the reorganization of the respository.
```

Comment:
would it be possible to keep this possibility in mind when organizing the repo in the first place? imo it's more likely than not that ~we~ you will extend this to other teams, and so if we develop with that in mind ahead of time it'll save trouble down the road even though it isn't necessary for an MVP.

Reply:
Hmmm I'm pretty against future proofing if it leads to more work. BUT I think what it would come to would be creating a new folder system and having the api reflect that. 

Which is why I'm a little hesitant to implement immediately with everything because then it would just add a layer of abstraction on smth that could be built faster?

Reply:
fair enough, i also do tend to overengineer. if you think the amount of work needed to extend down the line isn't significantly greater, then yeah i agree with you